automated-tests/src/main/java/com/hp/ov/sdk/network/FcoeNetworkResource.java

```diff
@@ -97,4 +104,27 @@ public FcoeNetwork builderUpdate(FcoeNetwork fcoeNetwork) {
         fcoeNetwork.setName(resourceProperties.get("name"));
         return fcoeNetwork;
     }
+
+	public String patchFcoeNetwork(String id) {
+		FcoeNetwork fcoeNetwork = new FcoeNetwork();
+		fcoeNetwork.setName(resourceProperties.get("name"));
+

```

Comment:
Please fix indentation for the 2 lines above.

Reply:
Done